src/server/viper.ts

```diff
@@ -9,27 +9,24 @@ import { Diagnostic, DiagnosticSeverity, Range } from 'vscode-languageserver';
 import { existsSync, unlinkSync, writeFileSync } from 'fs';
 import { sendDiagnostics } from './server';
 
-const serverPort = 54816; // TODO: choose automatically, or reuse server from Viper extension
-const viperServerPath = resolve(__dirname, '../generated/viperserver.jar'); // TODO: detect from Viper extension
+//const viperServerPath = resolve(__dirname, '../generated/viperserver.jar'); // TODO: detect from Viper extension
 const z3Path = resolve(__dirname, '../generated/z3'); // TODO: detect from Viper extension
 const verificationDebounce = 500; // TODO: config
 
 // Viper LSP server connection
 let connection: rpc.MessageConnection | undefined;
 
 try {
-    spawn(
-        'java',
+    const server = spawn(
+        '/nix/store/p4qnsh2pfcw444z0p0jji6rf0hl5r9wk-zulu17.34.19-ca-jdk-17.0.3/bin/java',
```

Comment:
restore this

Reply:
It fails with just `java`.

Reply:
The few commits leading up to 79b15439d1ffe04490afbd98a605ec97495012ed make me sound optimistic that we can pass configuration arguments into the `server.js`.